Makefile

```diff
@@ -1,13 +1,14 @@
 .PHONY: all integration-tests deployer geth-l2 batch-submitter data-transport-layer test
 
+pull:
+	@echo "Pulling & building all remote images..."
```

Comment:
`pull.sh` is broken so i don't think we should recommend using it until it is fixed